src/components/Toast/DefaultToast/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import styles from '@/components/Toast/DefaultToast/DefaultToast.css';
+import IconClose from '@/assets/images/icons/web/close.svg';
+
+interface Props {
+  className?: string;
+  autoClose?: number;
```

Comment:
maybe we could rename this to `autoCloseInMilliseconds` because right now it is hard to understand how to use this